tools/tuhi-live.py

```diff
@@ -201,6 +206,14 @@ def main(args=sys.argv):
         sys.exit('Script must be run as root')
 
     our_args, remaining_args = parse(args)
+    if our_args.flatpak_compatibility_mode:
+        from pathlib import Path
+
+        basedir = Path.home() / '.var' / 'app' / 'org.freedesktop.Tuhi'
+        print(f'Using flatpak xdg dirs in {basedir}')
+        os.environ['XDG_DATA_HOME'] = os.fspath(basedir / 'data')
+        os.environ['XDG_CONFIG_HOME'] = os.fspath(basedir / 'config')
+        os.environ['XDG_CACHE_HOME'] = os.fspath(basedir / 'cache')
```

Comment:
So, couple of things:
- `Path.home()` returns `/root`, which is not the sudo user -> we should use `Path(os.path.expanduser('~' + os.getlogin()))`
- `drop_privileges()` clears our environment, so all the work done here has been removed

Following patch works for me:
```patch
diff --git a/tools/tuhi-live.py b/tools/tuhi-live.py
index 61c6df1b4ee5..c06d33173ca4 100755
--- a/tools/tuhi-live.py
+++ b/tools/tuhi-live.py
@@ -206,14 +206,6 @@ def main(args=sys.argv):
         sys.exit('Script must be run as root')
 
     our_args, remaining_args = parse(args)
-    if our_args.flatpak_compatibility_mode:
-        from pathlib import Path
-
-        basedir = Path.home() / '.var' / 'app' / 'org.freedesktop.Tuhi'
-        print(f'Using flatpak xdg dirs in {basedir}')
-        os.environ['XDG_DATA_HOME'] = os.fspath(basedir / 'data')
-        os.environ['XDG_CONFIG_HOME'] = os.fspath(basedir / 'config')
-        os.environ['XDG_CACHE_HOME'] = os.fspath(basedir / 'cache')
 
     request_fd_queue = multiprocessing.Queue()
     conn_in, conn_out = multiprocessing.Pipe()
@@ -224,6 +216,16 @@ def main(args=sys.argv):
 
     drop_privileges()
 
+    if our_args.flatpak_compatibility_mode:
+        from pathlib import Path
+
+        basedir = Path(os.path.expanduser('~' + os.getlogin())) / '.var' / 'app' / 'org.freedesktop.Tuhi'
+        print(f'Using flatpak xdg dirs in {basedir}')
+        os.environ['XDG_DATA_HOME'] = os.fspath(basedir / 'data')
+        os.environ['XDG_CONFIG_HOME'] = os.fspath(basedir / 'config')
+        os.environ['XDG_CACHE_HOME'] = os.fspath(basedir / 'cache')
+
+
     start_tuhi_server(remaining_args)
     run_live(request_fd_queue, conn_in)
```